### DISCUSSION TITLE
Is there an anti cheat implementation in O3DE?

### DISCUSSION THREAD

**Discussion by archerallstars** [o3de/o3de#6769] [Q&A] [answered]
For Unreal, there's Easy Anti Cheat. What about O3DE, is there something similar?

Thanks in advance.

**Comment by lmbr-pip** [comment, marked as answer]
O3DE has a server authoritative networking architecture which prevents many forms of cheating. However O3DE does not currently offer parallel services like Epics "Easy Anti-Cheat" service.

Are there specific features you are looking for, asking as often anti-cheat means different things to different folks depending on genre/game design.

**Comment by soltsmank** [comment]
Hi,

There are a lot of possible cheats, as @lmbr-pip mentioned, it depends on a game.
Also, don't think it is a easy done, it is an endless loop between making cheats and anti-cheaters.

**Comment by archerallstars** [comment]
@lmbr-pip and @soltsmank

I am asking this out of a curiosity on wether to use O3DE in my new project. I don't have any specific need in mind (yet). However, having something to prevent a cheating on the client side is better than having none at all.

Thanks for your answers.

**Reply by soltsmank** [reply]
Thinking a while on a topic...
As the engine concept to be modular, it is good to have a several Gems for anti cheating enabled.
